estate/models/property.py

```diff
@@ -0,0 +1,48 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class Property(models.Model):
+    _name = 'estate.property'
+    _description = 'Estate property'
```

Comment:
Please use double quotes for non-technical strings
```suggestion
    _description = "Estate property"
```